logverification/app/massifgetteroptions.go

```diff
@@ -0,0 +1,66 @@
+package app
+
+import (
+	"github.com/datatrails/go-datatrails-common/azblob"
+	"github.com/datatrails/go-datatrails-merklelog/massifs"
+)
+
+/**
+ * Massif Options for the App are how the App Entries retrieve the correct massif to get data from their corresponding log entry.
+ */
+
+// MassifGetterOptions how an app entry retrieves its massif
+type MassifGetterOptions struct {
+	*MassifOptions
+
+	azblobReader azblob.Reader
+
+	massifReader MassifGetter
```

Comment:
calling something a Reader implies that it corresponds to the io.Reader interface which this does not.

why not massifGetter?

Reply:
good suggestion :)

Reply:
done!